### PR TITLE
[RFC] Switch remaining keyboard icons to glyphs

### PR DESCRIPTION
**Pull request by NiLuJe**
The only two left were Del/Backspace & Enter.

This also allows one to render a specific key in synthetic bold, in order to do so for those two glyphs.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/5639)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Before:
kb_icon

After:
kb_glyph

**Comment by Frenzie** [comment]
Thanks, I was just about to ask. ;-)

**Comment by NiLuJe** [comment]
The enter glyph is slightly smaller, hence the "bold" trick to compensate.

FWIW, unbolded:

kb_cr_bis

**Comment by NiLuJe** [comment]
Other candidates for the enter glyph (c.f., https://github.com/koreader/koreader/pull/5583#issuecomment-554717198):

* The actual Enter glyph, which is unfortunately super-weird in FreeSerif:
kb_enter

* CR:
kb_cr

* Nerdfont's Enter key:
kb_nerdf

**Comment by NiLuJe** [comment]
TL;DR: The only other decent one is probably nerdfont's ;).

I wasn't necessarily going to bold the Backspace, but it behaves slightly less worse that way re: A2/ghosting on eInk when you tap it ;).

**Comment by NiLuJe** [comment]
(gnu.org / savannah.org is down, hence the CI failures).

**Comment by poire-z** [comment]
Not fond of the bold backspace (our orignal icon looked a bit airy'er and nicer to me, but ok).
Really nothing else for backspace in nerdfont?

(No problem with the bolded or unbolded enter key.)

**Comment by NiLuJe** [comment]
I'll double-check, but, surprisingly, no. There's all the crappy Apple meta keys, but not that one -_-".

In case I don't find anything else, are you okay with that one unbolded?

**Comment by NiLuJe** [comment]
... he said before remembering there was the Material design one.

Which is decidedly NOT airy ;p.

kb_mdi

**Comment by NiLuJe** [comment]
The current MDI set *does* include an outline variant, though (https://materialdesignicons.com/), so I may be able to finagle something if that one's more amenable ;).

**Comment by poire-z** [comment]
> In case I don't find anything else, are you okay with that one unbolded?

I'll be ok :) Unbolded prefered, but hava a go with bold it it avoids that ghosting issue.

> Which is decidedly NOT airy ;p.

Yep, quite too solid black (it looks like that battery icon knocked down! :)

**Comment by NiLuJe** [comment]
Okay, that looks better with the outline variant ;).

kb_mdi_outline

**Comment by poire-z** [comment]
Way better ! :+1:
